src/server/vnparse.c

```diff
@@ -2432,6 +2434,55 @@ reliable_job_node_delete(pbs_list_head *node_list, char *nname)
 
 }
 
+/**
+ *
+ * @brief
+ *	Find an entry from the list of nodes for a job
+ *	named 'nname', and mark this node host as having
+ *	successfully executed execjob_prologue hook,
+ *	resulting in a hook event accept.
+ *	If no existing node host was matched, then add one.
+ *
+ * @param[in]	node_list	- the PBS node list
+ * @param[in]	nname		- node hostname to search for.
+ *
+ * @return reliable_job_node *
+ *
+ * @retval <reliable_nob_node entry>	- the updated/added node entry
+ * @retval NULL  			- if an error occurred.
+ * 
+ */
+reliable_job_node *
+reliable_job_node_set_prologue_hook_success(pbs_list_head *node_list, char *nname)
+{
+	reliable_job_node *rjn = NULL;
+
+	if ((node_list == NULL) || (nname == NULL))
+		return (NULL);
+
+	for (rjn = (reliable_job_node *)GET_NEXT(*node_list); rjn != NULL; rjn = (reliable_job_node *)GET_NEXT(rjn->rjn_link)) {
+		if (strcmp(rjn->rjn_host, nname) == 0) {
+			rjn->prologue_hook_success = 1;
+			return (rjn);
+		}
+	}
+	/* no entry matched so add one */
+	rjn = (reliable_job_node *)malloc(sizeof(reliable_job_node));
+	if (rjn == (reliable_job_node *)0) {
```

Comment:
Use NULL.  We just had a PR come through to remove all of these casted 0's

Reply:
Oh yes. Fixed. I also fixed similar instances througout vnparse.c